Lib/test/test_capi/test_misc.py

```diff
@@ -2215,16 +2215,17 @@ def test_configured_settings(self):
                 self.assertEqual(settings, expected)
 
         # expected to fail
-        for config in expected_to_fail:
-            kwargs = dict(zip(kwlist, config))
-            with self.subTest(config):
-                script = textwrap.dedent(f'''
-                    import _testinternalcapi
-                    _testinternalcapi.get_interp_settings()
-                    raise NotImplementedError('unreachable')
-                    ''')
-                with self.assertRaises(_interpreters.InterpreterError):
-                    support.run_in_subinterp_with_config(script, **kwargs)
+        if _interpreters is not None:
```

Comment:
Apologies for not picking up this one in my previous review - but could we skip this entire test with `@requires_subinterpreters`? If there's no subinterpreter module, I'm guessing L2201-2215 are only passing because there's no "expected to work" configurations

Reply:
Indeed that works.